codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala

```diff
@@ -107,39 +106,41 @@ object ActionServiceSourceGenerator {
       "package" <+> packageName <> semi <> line <>
       line <>
       ssep(
-        imports.map(pkg => "import" <+> pkg <> semi),
+        imports.to[immutable.Seq].map(pkg => "import" <+> pkg <> semi),
```

Comment:
kiama expected `immutable.Seq` for all methods, but when compiling with 2.12 we get the mutable one. 

Could have changed more code to always return immutable, but since we are stepping out of kiama, I don't think it's worth the effort.